riot/riot.py

```diff
@@ -25,6 +26,7 @@
 
 DEFAULT_RIOT_PATH = ".riot"
 DEFAULT_RIOT_ENV_PREFIX = "venv_py"
+SETUP_FILES = ["setup.py", "pyproject.toml"]
```

Comment:
`setup.cfg`?

Reply:
Good question. I think `pip` looks for just `setup.py` and/or `pyproject.toml`, but `setup.cfg` should probably be hashed as well, if present 🤔

Reply:
do we want to add `setup.cfg` here? probably a good idea since it can contain install/build requirements

... would that mean we also care about `requirements.txt`? or any variation of those?